Recast/Source/RecastAlloc.cpp

```diff
@@ -77,6 +78,7 @@ void rcIntArray::doResize(int n)
 	if (!m_cap) m_cap = n;
 	while (m_cap < n) m_cap *= 2;
 	int* newData = (int*)rcAlloc(m_cap*sizeof(int), RC_ALLOC_TEMP);
+	rcAssert(newData);
 	if (m_size && newData) memcpy(newData, m_data, m_size*sizeof(int));
```

Comment:
If we run out of memory in a non-debug build, we don't want things to crash.  Maybe we should indicate that we ran out of memory somewhere when this happens, rather than failing silently?